models/sinusoidal_gamma_generator.h

```diff
@@ -140,8 +147,8 @@ SeeAlso: sinusoidal_poisson_generator, gamma_sup_generator
  * are changed at
  *       \f$t_c > t_0\f$, then \f$\Lambda(t)\f$ is integrated piecewise for
  *       \f$t>t_c\f$ as
- *       @f[ \Lambda(t) = a_{old} \int_{t_0}^{t_c]} \lambda_{old}(s) ds
- *                      + a_{new} \int_{t_c}^{t]} \lambda_{new}(s) ds @f]
+ *        \Lambda(t) = a_{old} \int_{t_0}^{t_c]} \lambda_{old}(s) ds
+ *                      + a_{new} \int_{t_c}^{t]} \lambda_{new}(s) ds
```

Comment:
These are all inside of a Doxygen comment and should thus not be changed to RST syntax. Please revert.